portfolio/src/main/webapp/index.html

```diff
@@ -1,33 +1,43 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Karla&family=Montaga&family=Montserrat:wght@200&display=swap" rel="stylesheet">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="header_text">
-        <h1>Smruthi Balajee</h1>
-        <p>Welcome to my portfolio!</p>
+  <body onload="writeName()"> 
+      <div id="navbar">
+        <a href="#home">Home</a>
+        <a href="#about-me">About Me</a>
+        <a href="#contact">Contact</a>
     </div>
```

Comment:
nit: Fix the indentation so that this closing div tag lines up with its opening tag. Also I noticed 4 whitespace sand 2 whitespaces used in this file for indentation. Not sure if your editor can auto fix that to make it consistent.

Reply:
Got it, thanks! Just fixed the tabs + spaces issue and the indentation levels.